src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -157,6 +158,7 @@ const SettingsAccessLinkManagement = ({
                 description: 'Column header for the link management data table in the settings page.',
               }),
               accessor: 'link',
+              // eslint-disable-next-line react/no-unstable-nested-components
```

Comment:
nit: the `eslint-disable` no longer needed now that `Cell: LinkTableCell` is unchanged